js/game.js

```diff
@@ -735,11 +650,13 @@ class Level extends Scene {
 				screen_min = this.room_start;
 				screen_max = this.room_start + this.engine.canvas.width;
 
-				const e = new CoolScreen(
-					'BOSS TIME!',
+				const e = new OverlayEntity(
 					this.engine.canvas.width,
 					this.engine.canvas.height,
-					true,
+					{
+						Boss,
+						Grimey,
+					},
```

Comment:
`OverlayEntity` needs their constructors, this was a simple way to pass them along without having to do any imports (trying to avoid circular dependencies)